freqtrade/rpc/rpc.py

```diff
@@ -226,6 +226,53 @@ def _rpc_daily_profit(
             for key, value in profit_days.items()
         ]
 
+    def _rpc_trade_history(
+            self, last_trades_number: int) -> Dict[str, List[Dict[str, Any]]]:
+        """ Returns the X last trades """
+        if last_trades_number > 0:
+            trades = Trade.get_trades().order_by(Trade.id.desc()).limit(last_trades_number)
+        else:
+            trades = Trade.get_trades().order_by(Trade.id.desc()).all()
+
+        output = []
+
+        for trade in trades:
+            output.append({
```

Comment:
I would rely on `trade.to_json()` for this to have one common "trade to json conversion" method.

It implements part of what's needed here ... so you may need to extend that function to return "everything", but that shouldn't be a problem for anything else (more is always better - no)?

Reply:
Yes better to have an unified trade view